api/src/main/java/org/openmrs/ConditionClinicalStatus.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs;
+
+/**
+ * The ConditionClinicalStatus is what we've historically called "condition status" and what FHIR refers to as
+ * ConditionClinicalStatus . This is a clinical condition that has risen to a level of concern.
+ * 
+ * <li>{@link #ACTIVE}</li>
+ * <li>{@link #INACTIVE}</li>
```

Comment:
How do these three lines (17 - 19) connect to the above?

Reply:
The three lines directly link to the documentation for the defined values in the enum which improves readability of the Javadoc

Reply:
Did you see how this one did it? https://github.com/openmrs/openmrs-core/commit/7411f5cce4eba1dbc83a89de699f240d124c5502

Reply:
Alright, let me refactor accordingly